### DISCUSSION TITLE
How to write/read plane vars?

### DISCUSSION THREAD

**Discussion by MeinTollerUsername** [flybywiresim/aircraft#4518] [Q&A] [answered]
Dear FlyByWireSim Community,

i love to fly the A32nx in the MSFS and im amazed about the process going on here. 
Im building a A320 single-pilot cockpit and i want to integrate my cockpit-parts to the A320.
Is there currently a way to do this? 
Are there some Event-IDs which can be read/write? 
Is there a kind of documentation about it? 

At the moment im using [Mobiflight](https://www.mobiflight.com/de/index.html) to read/write values from the sim.

If there are any ways to write/read values with an Arduino please let me know. 
My plan is to integrate cockpit parts to the a32nx.

Another question is if the mcdu & efb screen can be "released" to another external monitor? 

Thank you so much for your great work!

Jonas

**Comment by ExampleWasTaken** [comment]
I don't think there is a way to connect a cockpit to the aircraft. 

To answer your question about the EFB: You can pop all screen out with `RIGHT ALT+CLICK`. It is not currently possible to display them on another device.

**Reply by MeinTollerUsername** [reply]
Thanks for your answer. 

The question is of the airplane is accessable via the sim variables

**Comment by tracernz** [comment, marked as answer]
[Jetbridge](https://github.com/theomessin/jetbridge) is what I intend to use. Many of the event bars are documented in the experimental branch, which will in future become stable.

**Reply by MeinTollerUsername** [reply]
Looks interesting. 
Do you have experience with that?

**Reply by tracernz** [reply]
> 
> 
> Looks interesting.
> Do you have experience with that?

Not much, but it's on my eventual todo list.
